### PR TITLE
removed weird call of Class.getClass()

### PR DESCRIPTION
**Pull request by dk2k**
removed weird call of Class.getClass()

### CODE REVIEW

**Review by essobedo — COMMENTED** [review]
Sounds good, but could you please modify your commit message for something more clear?

**Review by orpiske — CHANGES_REQUESTED** [review]
LGTM, but please adjust the commit message.

**Review by oscerd — CHANGES_REQUESTED** [review]
Commit message is useless

File: tooling/maven/camel-package-maven-plugin/src/main/java/org/apache/camel/maven/packaging/generics/GenericsUtil.java [hunk 1]
```diff
@@ -74,7 +74,7 @@ private static boolean isAssignableFrom(boolean isDelegateOrEvent, Class<?> inje
         } else if (beanType instanceof WildcardType) {
             return isAssignableFrom(isDelegateOrEvent, (Type) injectionPointType, (WildcardType) beanType);
         } else {
-            throw new IllegalArgumentException("Unsupported type " + injectionPointType.getClass());
+            throw new IllegalArgumentException("Unsupported type " + injectionPointType);
```
[the remark this answers is not in the document]
Comment (dk2k) [reply tooling/maven/camel-package-maven-plugin/src/main/java/org/apache/camel/maven/packaging/generics/GenericsUtil.java:77]: in the line 61 requiredType has type implementing Type (it's an interface). Hence, requiredType.getClass() gives relevant information

**Comment by orpiske** [comment]
Not trying to be pedantic, but please try something like this for the commit message: 

`Removed a call of Class.getClass(). getClass() that is called on the instance of type Class<?> and additional getClass() call that could erase the type'

Please, make sure to **rebase** so it rewrites the history and avoids writing the messages (where `upstream` is our  repo https://github.com/apache/camel and `origin` is yours). Something like this should do the trick:

`git pull --rebase upstream main; git checkout <my-working-branch> ; git rebase main ; git push -f origin <my-working-branch>`

**Comment by dk2k** [comment]
@orpiske I appreciate your time for sending git commands (I saved them just in case), but it's still a rocket science to me. I have an IDE with comfortable menus and console git is beyond my comfort zone. The changes here are pretty straightforward, so I rebased the PR itself :)
new PR https://github.com/apache/camel/pull/9812

**Comment by davsclaus** [comment]
closing this

**Pull request closed without merging**